vendor/golang.org/x/mobile/app/app.go

```diff
@@ -140,7 +140,6 @@ func (a *app) HideVirtualKeyboard() {
 	driverHideVirtualKeyboard()
 }
 
-
```

Comment:
p.s. best not to commit format changes in vendor files - they will just get overwritten by the next vendor update so it's just noise